editoast/editoast_authz/src/authorizer.rs

```diff
@@ -277,5 +367,26 @@ mod tests {
                 .collect();
             Ok(removed_roles)
         }
+
+        async fn get_user_id(&self, user_info: &UserInfo) -> Result<Option<i64>, Self::Error> {
+            self.users
+                .lock()
+                .unwrap()
+                .get(&user_info.identity)
+                .map(|id| Ok(Some(*id)))
+                .unwrap_or_else(|| Ok(None))
```

Comment:
Isn't this equivalent to this?

```suggestion
            Ok(self.users
                .lock()
                .unwrap()
                .get(&user_info.identity)
                .copied())
```